workplace/webview.js

```diff
@@ -28,10 +29,38 @@ module.exports = (Franz, options) => {
     }
   })(XMLHttpRequest.prototype.send)
 
+  function highlightMentions() {
+    //sometimes the chatlog is not rendered when this happens
+    try {
+      var messages = document.querySelector('[aria-label="Messages"]').querySelectorAll('[data-tooltip-position="left"] span')
+      var regex = new RegExp("(.*)" + name + "(.*)", "i");
+      var msg;
+      for(var i = 0; i < messages.length; i++) {
+        msg = messages[i];
+        if(regex.test(msg.innerHTML)) {
+          msg.classList.add('f-mention');
+        }
+      }
+    } catch(e) {}
+  }
+
+  // Different init times due to render variations, need to make sure the elements exist
+  function initHighlight() {
+    var chatLog = document.querySelector('div[role="presentation"]');
+    if (chatLog) {
+      name = document.querySelector('[title=Profile] span').innerText.toLowerCase();
+      observeDom(chatLog, highlightMentions);
+      highlightMentions();
+      window.clearInterval(interval);
+    }
+  }
 
   window.addEventListener('focus', function (e){
     Franz.setBadge(unread = 0)
   }, false);
+  window.addEventListener('load', function(){
+    interval = window.setInterval(initHighlight, 150);
+  });
```

Comment:
I think you could use `Franz.loop(fn)` here instead of your own timer. That considered, do we really need the dom mutation observers if this is running in a loop?